aws/FSS-user-permissions.json

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
Can we rename this file to `deployment-role-policy.json `?

Reply:
Done!